src/OpenTelemetry/Metrics/BaseExportingMetricReader.cs

```diff
@@ -21,7 +21,6 @@ namespace OpenTelemetry.Metrics
     public class BaseExportingMetricReader : MetricReader
     {
         protected readonly BaseExporter<Metric> exporter;
-        protected bool disposed;
```

Comment:
Follow up on https://github.com/open-telemetry/opentelemetry-dotnet/pull/2342#discussion_r708810437.

I've moved `disposed` to the base class and made it private.
I guess I'll leave `exporter` as "protected readonly", this makes the code consistent with https://github.com/open-telemetry/opentelemetry-dotnet/blob/34d8332a809b6258f481fdd07deea1b52b5b080d/src/OpenTelemetry/BaseExportProcessor.cs#L29.

Reply:
I don't have an issue with this just want to share the CA warning doc which has a bit of the reasoning behind not exposing fields: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1051

TL;DR: The primary use of a field should be as an implementation detail.